src/Pages/Login/Components/Login.js

```diff
@@ -0,0 +1,126 @@
+import styles from "./Login.module.css";
+import axios from "axios";
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import { useAuthData } from "../../../Context/AuthContext";
+import { useNavigate } from "react-router-dom";
+
+const LoginForm = () => {
+  const navigate = useNavigate();
+
+  const [login, setLogin] = useState({
+    email: "",
+    password: "",
+    viewPassword: false,
+    error: "",
+  });
+
+  //Auth Context
+  const { DispatchUserAuth } = useAuthData();
+
+  // destructure login variable
+  const { email, password, viewPassword, error } = login;
+
+  // set error
+  const setError = (msg) => {
+    setLogin((prev) => ({
+      ...prev,
+      error: msg,
+    }));
+  };
+
+  // submit handler
+  const submitHandler = async (e) => {
+    e.preventDefault();
+    console.log("jana hai");
+    try {
+      const response = await axios.post(`/api/auth/login`, {
+        email,
+        password,
+      });
+      console.log("ane do");
+      console.log(response);
+
+      if (response.status === 200 || response.status === 201) {
+        console.log(response);
```

Comment:
remove it

Reply:
Thanks for pointing it out, fixed it in the latest commit